glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/giantswarm/crdstorage
 import:
 - package: github.com/giantswarm/apiextensions
-  version: release-1.8
+  version: clientset
```

Comment:
That did not work out apparently. Needed to go back to the other branch. We have to live with this right now. Looking forward to next year when we can finish all the migration magic and get this straight everywhere.